Fix building on PIC hosts (filename.o was added to LIBOBJECTS_X but this
target wasn't updated ... so it gets built w/out -fPIC)

--- lib/Makefile
+++ lib/Makefile
@@ -184,7 +184,7 @@
 compile.h:
 	$(SRCDIR)/buildtools/stamp-date >$@ || rm $@
 
-util/shhopt.o util/nstring.o: FORCE
+$(LIBOBJECTS_X): FORCE
 	@if [ ! -d $(dir $@) ] ; then mkdir $(dir $@) ; fi
 	$(MAKE) -C $(dir $@) -f $(SRCDIR)/$(SUBDIR)/$(dir $@)Makefile \
 		SRCDIR=$(SRCDIR) BUILDDIR=$(BUILDDIR) $(notdir $@) 
